OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -351,13 +354,14 @@ private void populatePostList(@NonNull final InfiniteScrollUtils.PageLoadCallbac
         }
 
         final Activity activity = getActivity();
-        final TaskMessageCallback mCallback = activity instanceof TaskMessageCallback ? (TaskMessageCallback) activity : null;
         final boolean isRefreshingSilently = callback.isRefreshingSilently();
         getThreadListCall.enqueue(new ErrorHandlingCallback<Page<DiscussionThread>>(activity,
                 // Initially we need to show the spinner at the center of the screen. After that,
                 // the ListView will start showing a footer-based loading indicator.
                 nextPage > 1 || isRefreshingSilently ? null :
-                        new ProgressViewController(loadingIndicator),mCallback, CallTrigger.LOADING_UNCACHED) {
+                        new ProgressViewController(loadingIndicator),
+                // We only require the error to appear if the first server call fails
```

Comment:
How user will aware of the error if it comes in second or plus call ?

Reply:
Generally, we should show a SnackBar when the user encounters an error after the first load (which is out of scope currently). For now, we won't be showing the pagination loader and displaying whatever was data loaded in the first call.